.rubocop.yml

```diff
@@ -70,6 +70,9 @@ Rails/ApplicationController:
     - 'app/controllers/platform_controller.rb'
     - 'app/controllers/public_controller.rb'
     - 'app/controllers/survey/responses_controller.rb'
+Rails/SkipsModelValidations:
+   Exclude:
+    - 'app/services/automation_rules/action_service.rb'
```

Comment:
Added this in order to user update_column here: https://github.com/chatwoot/chatwoot/pull/4087/files#r816964170